lib/modules/web3/transports.ts

```diff
@@ -17,3 +17,8 @@ export function getTransports(chain: Chain) {
 export const transports = Object.fromEntries(
   chains.map(chain => [chain.id, getTransports(chain)])
 ) as Record<number, ReturnType<typeof getTransports>>
+
+export function getHandlerRpcUrl(chainId: number): string {
```

Comment:
Why does it need to be called `handlers`, I don't see why that context needs to be included here.
```suggestion
export function getRpcUrl(chainId: number): string {
```

Reply:
I was hesitating and decided to be explicit cause only the handlers should be using this helper but that's implicit in the code so I will rename it as you suggest.